kafka-ui-react-app/src/components/Schemas/Edit/Edit.styled.ts

```diff
@@ -30,14 +30,18 @@ export const EditorsWrapper = styled.div`
 
   & > * {
     flex-grow: 1;
-    border: 1px solid #e3e6e8;
-    border-radius: 8px;
-    padding: 16px;
-    & > h4 {
-      font-weight: 500;
-      font-size: 16px;
-      line-height: 24px;
-      padding-bottom: 16px;
-    }
+  }
+`;
+
+export const EditorsWindow = styled.div`
```

Comment:
```suggestion
export const EditorWindow = styled.div`
```

Besides, I think it would be better to call it EditorContainer